pkgs/by-name/gk/gk-cli/package.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  coreutils,
+  installShellFiles,
+  makeWrapper,
+}:
+
+let
+  version = "2.1.0";
+in
+stdenv.mkDerivation {
+  pname = "gk-cli";
+  inherit version;
+
+  src = fetchzip {
+    url = "https://github.com/gitkraken/gk-cli/releases/download/v${version}/gk_${version}_Linux_x86_64.zip";
+    sha256 = "sha256-BZO8MVFxIXXB+9v4yB9XcZ0E68xOIUL7Mot7PZ+5M1U=";
+    stripRoot = false;
+  };
+
+  nativeBuildInputs = [
+    installShellFiles
+    makeWrapper
+  ];
+
+  postInstall = ''
+    mkdir --parents $out/bin/
+    install ./gk $out/bin/
+
+    installShellCompletion --bash ./**/gk.bash
+    installShellCompletion --fish ./**/gk.fish
+    installShellCompletion --zsh ./**/_gk
+  '';
+
+  postFixup = ''
+    wrapProgram $out/bin/gk \
+      --prefix PATH : "${lib.makeBinPath [ coreutils ]}"
+  '';
+
+  meta = {
+    description = "GitKraken CLI is the ultimate CLI for Git collaboration";
```

According to the [recommendations for meta attributes](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes) description should avoid mentioning the package and be short succinct:
```suggestion
    description = "CLI for Git collaboration across multiple repos and services";
```